core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -59,6 +64,36 @@ public interface Configs {
           .checkValue(StringUtils::isNotBlank, ConfigConstants.NOT_BLANK_ERROR_MSG)
           .createWithDefault(DEFAULT_ENTITY_RELATIONAL_STORE);
 
+  ConfigEntry<String> ENTRY_RELATIONAL_MYSQL_BACKEND_URL =
+      new ConfigBuilder(MYSQL_ENTITY_STORE_URL_KEY)
+          .doc("Connection URL of `MySQLBackend`")
+          .version("0.5.0")
+          .stringConf()
+          .checkValue(StringUtils::isNotBlank, ConfigConstants.NOT_BLANK_ERROR_MSG)
+          .create();
+
+  ConfigEntry<String> ENTRY_RELATIONAL_MYSQL_BACKEND_DRIVER_NAME =
+      new ConfigBuilder(MYSQL_ENTITY_STORE_DRIVER_NAME_KEY)
+          .doc("Driver Name of `MySQLBackend`")
+          .version("0.5.0")
+          .stringConf()
+          .checkValue(StringUtils::isNotBlank, ConfigConstants.NOT_BLANK_ERROR_MSG)
+          .create();
+
+  ConfigEntry<String> ENTRY_RELATIONAL_MYSQL_BACKEND_USERNAME =
+      new ConfigBuilder(MYSQL_ENTITY_STORE_USERNAME_KEY)
+          .doc("Username of `MySQLBackend`")
+          .version("0.5.0")
+          .stringConf()
```

Comment:
Could you add some documents for these config options in the document https://github.com/datastrato/gravitino/blob/main/docs/gravitino-server-config.md?

1. Prefer this config option without default value.
2. Prefer adding checkValue for this option.